docs/beta/tutorial/continuous_mastering.md

```diff
@@ -0,0 +1,111 @@
+To complete this tutorial you will need:
+- `tamr-unify-client` [installed](../../user-guide/installation)
+- access to a Tamr instance, specifically:
+  - a username and password that allow you to log in to Tamr
+  - the socket address of the instance
+- an existing Mastering project with loaded data that has been run to the end at least once
+  - it is recommended that you first complete the tutorial [here](https://docs.tamr.com/tamr-tutorials/docs/overview-mastering)
+  - alternatively, a different Mastering project can be used however the project name will likely be different
+
+## Steps
+### Configure the Session and Instance
+- Use your username and password to create an instance of `tamr_client.UsernamePasswordAuth`.
+- Use the function `tamr_client.session.from.auth` to create a `Session`.
+```eval_rst
+.. literalinclude:: ../../../examples/continuous_mastering.py
+    :language: python
+    :lines: 1-9
+```
+- Create an `Instance` using the `protocol`, `host`, and `port` of your Tamr instance. Replace the values of `protocol`, `host`, and `port` with the corresponding values for your Tamr instance.
+```eval_rst
+.. literalinclude:: ../../../examples/continuous_mastering.py
+    :language: python
+    :lines: 11-15
+```
+
+### Get the Tamr Mastering project to be updated
+Use the function `tc.project.by_name` to retrieve the project information from the server.
+```eval_rst
+.. literalinclude:: ../../../examples/continuous_mastering.py
+    :language: python
+    :lines: 17
+```
+Ensure that the retrieved project is a Mastering project by checking its type:
+```eval_rst
+.. literalinclude:: ../../../examples/continuous_mastering.py
+    :language: python
+    :lines: 19-20
+```
+
+### Update the unified dataset
+To apply the [attribute mapping configuration](https://docs.tamr.com/tamr-tutorials/docs/define-project-schema-mastering) and any transformations to update the unified dataset with updated source data, use the function `tc.mastering.update_unified_dataset`.
+```eval_rst
+.. literalinclude:: ../../../examples/continuous_mastering.py
+    :language: python
+    :lines: 22-23
+```
+This function and all others in this tutorial are *synchronous*, meaning that they will not return until the job in Tamr has resolved, either successfully or unsuccessfully.  The function `tc.operation.check` will raise an exception and halt the script if the job started in Tamr fails for any reason.
+
+### Generate pairs
+To generate pairs according to the [configured pair filter rules](https://docs.tamr.com/tamr-tutorials/docs/setup-how-pairs-are-found), use the function `tc.mastering.generate_pairs`
+```eval_rst
+.. literalinclude:: ../../../examples/continuous_mastering.py
+    :language: python
+    :lines: 25-26
+```
+
+### Train the model with new Labels
+To [update the machine-learning model](https://docs.tamr.com/tamr-tutorials/docs/help-tamr-learn-about-your-data) with newly-applied pairs. use the function `tc.mastering.apply_feedback`
```

Comment:
That's right, this should say "newly-applied labels".  It is not expected that the user apply new labels to project via the UI during this tutorial, but that the process described here will update the project based on any new labels that have been applied since the last time the model was trained.  If no new labels have been applied and the input datasets haven't changed, this will all run successfully, but should not cause any changes to occur.